app/views/proposals/_encouragement_text.html.haml

```diff
@@ -20,5 +20,10 @@
     %span.notranslate
       = @conference.title
     will only be as good as the content you present. Submit early, submit often!
+    %br
+    %b
+      Description:
+    %span
+      = @program.cfp.description
```

Comment:
Let's keep the description visible.
Do you think it makes sense to show it *before* the encouragement text? 
Encouragement text is also visible in other places (eg new form), let's show the description there as well.

Reply:
Yes that makes sense. So shall I make another partial and show it after the heading,(be it show or new)

Reply:
Either a partial, or you just add one line in the places where we want the description to show up.
Do you think a partial is really necessary to show just one field?

Reply:
Okay I'll make the necessary changes.